libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -12,6 +12,144 @@
 logger = logging.getLogger(__name__)
 
 
+class AzureOpenAIWhisperParser(BaseBlobParser):
+    """Transcribe and parse audio files.
+
+    Audio transcription is with Azure OpenAI Whisper model.
+    This is different to the Open AI Whisper parser and requires
```

Comment:
Do we need to mention here that although it's different than OpenAI, we're still using the openai SDK (and is therefore still a dependency)? It might also be worth elaborating how it's different to Open AI - for example, behavioural differences, payload formatting etc